circe/src/main/scala/circe.scala

```diff
@@ -0,0 +1,23 @@
+package memeid.circe
+
+import scala.util.Try
+
+import memeid.UUID
+import memeid.JavaConverters._
+
+import java.util.{ UUID => JUUID }
+
+import cats.syntax.show._
+
+import io.circe.{ Encoder, Decoder }
+
+
+object implicits {
```

Comment:
Why not creating them from `java.util.UUID`'s `Encoder`/`Decoder`? Like you did with doobie's `Get`/`Put`

Reply:
Good point, updated!